<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="editHost">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="history">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>History</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="commandsResponse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Commands response:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT TO DEVICE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandInput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Input command</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="voltageInput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amperageInput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Amperage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendCommandButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND COMMAND</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkErrorsButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CHECK ERRORS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TURN ON</string>
    </property>
   </widget>
   <widget class="QLabel" name="expectedVoltageLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>410</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Expected voltage: -</string>
    </property>
   </widget>
   <widget class="QLabel" name="expectedAmperageLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Expected amperage: -</string>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>251</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="amperageRadioButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>270</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Amperage</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="voltageRadioButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>300</y>
      <width>99</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setDefaulButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set default host and port</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentAmperageLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current amperage: -</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentVoltageLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current voltage: -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getAmperageButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>441</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GET AMPERAGE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getVoltageButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>GET VOLTAGE</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
